tests/conftest.py

```diff
@@ -199,6 +199,24 @@ def nbrhosts(ansible_adhoc, testbed, creds):
         devices[k] = EosHost(ansible_adhoc, "VM%04d" % (vm_base + v['vm_offset']), creds['eos_login'], creds['eos_password'])
     return devices
 
+@pytest.fixture(scope="module")
+def fanouthosts(ansible_adhoc, creds):
+    """
+    Shortcut fixture for getting Fanout host
+    """
+    devices = {}
+
+    # TODO: Create a new module for get lab_graph
+    lab_graph = Parse_Lab_Graph(filename)
```

Comment:
This is a TODO.
Parse_Lab_Graph was used by conn_graph_facts module, but the conn_graph_facts only provide a single device information. To create fanouthosts fixture, we need to find all fanout switches.

The lab_graph includes all deviecs' hostname/type/mgmtip/...

Reply:
Why do you need all fanout switches? Can you get connection graph of the target DUT?

Reply:
This is for create a fanout hosts shortcut(pytest fixture). To import a fixture to a testcase, we need to index all fanout switches in the fixture just like nbrhosts which indexed all VM hosts.
Once all fanout hosts been indexed, we can use DUT's connection to get 1 of fanout hosts by need